<?xml version="1.0"?>

<launch>
  
    <param name="num_blocks" value="4" />
    <param name="configuration" value="stacked_ascending" />
    <param name="dual_arm_mode" value='True' />
    <param name="current_mode" value="scatter" />

    <node name="robot_interface" pkg="njllrd_proj1" type="robot_interface.py"
        output="screen" />
  
    <node name="controller" pkg="njllrd_proj1" type="controller.py" 
        output="screen" />
    
  
</launch>
